pom.xml

```diff
@@ -430,15 +431,25 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>${dependency-check-maven.version}</version>
```

Comment:
❓ The version is defined in the dependencyManagement section of ddf-parent, right?

Reply:
Great question. Looks like I made a mistake here, but it still works. I'll put the version back in. It seems like it grabs the latest version of dependency-check (5.2.4) and runs it just fine. There would be a pro to getting the latest, it just wouldn't be stable. Currently we are using 5.2.2 so might be time to upgrade again